src/ros_filter.cpp

```diff
@@ -733,10 +733,10 @@ namespace RobotLocalization
     bool stampedControl = false;
     double controlTimeout = sensorTimeout;
     std::vector<int> controlUpdateVector(TWIST_SIZE, 0);
-    std::vector<double> accelerationLimits(TWIST_SIZE, 0.0);
-    std::vector<double> accelerationGains(TWIST_SIZE, 0.0);
-    std::vector<double> decelerationLimits(TWIST_SIZE, 0.0);
-    std::vector<double> decelerationGains(TWIST_SIZE, 0.0);
+    std::vector<double> accelerationLimits(TWIST_SIZE, 1.0);
```

Comment:
Set defaults to `1.0`, not `0.0`, according with https://github.com/cra-ros-pkg/robot_localization/blob/kinetic-devel/params/ekf_template.yaml#L200 and the WARN messages from below, when the size is small, or the param is omitted.